packages/components/src/Link/Link.tsx

```diff
@@ -18,7 +18,10 @@ export function Link({
   return (
     <a
       href={url}
-      {...(external && { target: "_blank" })}
+      {...(external && {
```

Comment:
What are your thoughts on updating Button external links as well?
https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Button/Button.tsx#L57

Reply:
Good callout, I can do that!

Reply:
Done here b1df2c412b2dca9edbab8214da0ef607f0100812